### PR TITLE
Scale border size in window size pos calculation

### PR DESCRIPTION
**Pull request by Dickby**
#### Describe your PR, what does it fix/add?
Fixes https://github.com/hyprwm/Hyprland/issues/718

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
Don't see any

#### Is it ready for merging, or does it need work?
I think it's ready.

### CODE REVIEW

**Comment by fufexan** [comment]
I've built this but I see no difference in border size.

**Comment by Dickby** [comment]
The border size shouldn't change, and now the window gaps don't change either (on scale change).
Did you read the issue?

**Comment by Dickby** [comment]
I closed a pull request yesterday that did it the other way around.
By scaling the bordersize inside the renderer.
But i think this solution is nicer because window size and border size now are consistent on scale changes.
@vaxerski : if prefer the first solution (would feel more like actual monitor scaling, this one is more window scaling)
i will change it again.

**Comment by Estebiu** [comment]
How do you build merge requests? 
I'd like to try it.
Unrelated, but cool merge request number (727)

**Comment by Dickby** [comment]
@Estebiu you could clone and build my fork.
Probably there is a nicer way though.

**Comment by Dickby** [comment]
That test commit was not planned to be part of the pull request,
i didn't made a branch for the pr and now i just tried to push from my machine with a new gh token.
But i think it doesn't hurt.

**Comment by fufexan** [comment]
Took a look at the issue and tested again, seems to work fine.

If you could squash that last commit or rename it, that'd be great.

**Comment by Dickby** [comment]
Done!
nobody should give me write access until i really figured out git/github !

**Review by fufexan — APPROVED** [review]
LGTM. I'll let Vaxry comment or merge.

File: src/render/OpenGL.cpp
Comment (vaxerski) [inline src/render/OpenGL.cpp:1127]: wot

Comment (Dickby) [reply src/render/OpenGL.cpp:1127]: should i take that out? It was the result of commit to test the gh token i just made.
Didn't know that it will change the pull request.
Shouldn't change the behavior, and if unoptimized, it should be a bit more efficient.

Comment (Dickby) [reply src/render/OpenGL.cpp:1127]: I changed the 2.f to 2.0 since all the values are doubles and cairo_move_to takes doubles too.

Comment (Dickby) [reply src/render/OpenGL.cpp:1127]: By the way, am i supposed to resolve a conversation like this?

Comment (vaxerski) [reply src/render/OpenGL.cpp:1127]: remove this change. it's unrelated to the pr

**Pull request merged**
